pkgconfig/lcmtypes_bot2-core.pc.in

```diff
@@ -1,12 +1,13 @@
 prefix=@BOT2_CORE_INSTALL_PREFIX@
 exec_prefix=${prefix}
 libdir=${exec_prefix}/lib@LIB_SUFFIX@
-includedir=${prefix}/include
+includedir=${prefix}/include/
+includedir_lcmtypes=${prefix}/include/lcmtypes/
 
 Name: lcmtypes_bot2-core
 Description: LCM types for bot2-core
 Requires:  lcm
 Version: 0.0.0
 Libs: -L${libdir} -llcmtypes_bot2-core
-Cflags: -I${includedir}
+Cflags: -I${includedir} -I${includedir_lcmtypes}
```

Comment:
Cool. I also made a similar change in [RobotLocomotion/lcmtypes](https://github.com/RobotLocomotion/lcmtypes), although RL is using only `include/lcmtypes` rather than both.